### DISCUSSION TITLE
【委托】哔哩哔哩怎么把视频下载到手机上啊？

### DISCUSSION THREAD

**Discussion by runianqianhui** [share121/inter-knot#4966] [委托]
有没有好心人知道怎么把哔哩哔哩上面的视频下载到手机上？谢谢

**Comment by shuiren777** [comment]
_在手机文件管理里面找文件夹，但是b站的视频下载之后好像音频和视频是分开的_

**Comment by HEW666** [comment]
可以百度或者bing搜索 B站视频解析下载

**Comment by cyq20091029** [comment]
https://snapany.com/zh/bilibili
<img src="https://img-static.mihoyo.com/communityweb/upload/39f7ce769ba2902dbfee73517f7eec49.png" alt="妮可 奸笑" width="75" >


可以试试这个网站

**Comment by manyshoes** [comment]
网页端有下载脚本
